app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -217,9 +221,19 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
     };
 
     try {
-      await onWeeklyPickChange(params);
-      setUserPick(teamSelect);
-      router.push(`/league/${league}/entry/all`);
+      if (lockedOut) {
```

Comment:
Do we need to nest this if statement in the try block?

Reply:
Yes. So what it does is run the api functions inside the try block. But, if its during the lockdown period it will block the api functions from running and instead run the toast notification. It is a way to stop hackers from hitting the api's when they shouldn't.

Reply:
@braydoncoyer please let me know if you need more info on this

Reply:
@alexappleget @ryandotfurrer Yep, I get the purpose of the Try block. I think I was more asking if the try block could be moved _inside_ the conditional to better represent which part of the code may cause/throw an error. 


```tsx
if (lockedOut) {
  ... code
} else {
  try {
    await onWeeklyPickChange(params);
    setUserPick(teamSelect);
    ...
  } catch (error) {
   ...
  }
  console.error(params);
}
```